file_mapper_script.py

```diff
@@ -94,8 +94,36 @@ def get_parser():
                         directories can be moved without the link breaking.
                         """)
 
+    parser.add_argument('-sc', '--sidecars', dest='sidecars',
+                        required=False, default=False, action='store_true',
+                        help="""Also copy/move/symlink the corresponding
+                        json sidecars""")
+
     return parser
 
+
+def map_sidecars(source, destination):
+    ''' Return name of json sidecars to be copied/symlinked etc
+
+    Args:
+        source (str): full path to source file
+        destination (str): full path to destination
+    Returns:
+        json_src (str): full path to source json file
+        json_dest (str): fill path to destination json file
+    '''
+
+    replace = [".nii.gz", ".nii"]
+
+    for repl in replace:
+        if repl in source:
```

Comment:
I recommend using `source.endswith(repl)` here if the intent that those extensions would always be at the end.

Reply:
Yes, those are the file extensions so will always be at the end. The code has been updated (0b95714)